packages/@aws-cdk/aws-codepipeline-api/package.json

```diff
@@ -20,7 +20,8 @@
         "signAssembly": true,
         "assemblyOriginatorKeyFile": "../../key.snk"
       },
-      "sphinx": {}
+      "sphinx": {},
+      "python": {}
```

Comment:
Shouldn't this have the python names and stuffs?

Reply:
The Python target infers the name from the TypeScript name.

Reply:
Thinking about it a bit more, this seems dangerous to me.

What if we want to break the inferred package name derivation? How will we deal with package name sniping?

Does it at least provide an override? (If not, please add it today, because if I don't want to need to add this when we're in a high-stress situation later on...)

Reply:
+1 

We can’t have the language specific names be inferred. Let’s use the same pkglint mechanism to devise the python names.

Reply:
Ok, these no longer infer the names.